ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/pipes/noData.pipe.ts

```diff
@@ -6,8 +6,6 @@ import {Pipe, PipeTransform} from '@angular/core';
 
 export class NoDataPipe implements PipeTransform {
   transform(value: any): string {
-    const tempElement = document.createElement('div');
-    tempElement.innerHTML = value || '&nbsp;';
-    return tempElement.innerText;
+    return (value === 0 ? value.toString() : value) || String.fromCharCode(160);
```

Comment:
Would you add ` const nonBreakingSpace = String.fromCharCode(160);`?
It is not obvious from the first sight.

Reply:
Done!